DESCRIPTION

```diff
@@ -61,7 +61,9 @@ Imports:
     knitr (>= 1.12),
     yaml (>= 2.1.5),
     htmltools (>= 0.2.4),
-    caTools
+    caTools,
```

Comment:
We'll need to add these (and their dependencies if any) to the rmarkdown dependencies here: https://github.com/rstudio/rstudio/blob/master/src/gwt/src/org/rstudio/studio/client/common/dependencies/DependencyManager.java#L159-L175

Note that we've already got caTools as a downstream dependency (perhaps for knitr?). 

We depend on jsonlite for Shiny here: https://github.com/rstudio/rstudio/blob/master/src/gwt/src/org/rstudio/studio/client/common/dependencies/DependencyManager.java#L268 but we'd still need to add it for rmarkdown. If possible let's list the same version number (0.9.16) for the sake of consistency.